kubernetes/cmsweb/services/t0_reqmon-tasks.yaml

```diff
@@ -76,11 +76,11 @@ spec:
         name: t0reqmon-tasks
         resources:
           requests:
-            memory: "512Mi"
-            cpu: "200m"
+            memory: "750Mi"
+            cpu: "300m"
           limits:
-            memory: "5Gi"
-            cpu: "1500m"
+            memory: "2Gi"
```

Comment:
May be increase it to 3Gi. I have seen a spike upto 3Gi in the last month's dashboard